packages/jest-runtime/src/__tests__/__snapshots__/transform-test.js.snap

```diff
@@ -1,25 +1,25 @@
 exports[`transform transforms a file properly 1`] = `
-"({\"Object.<anonymous>\":function(module,exports,require,__dirname,__filename,global,jest){module.exports = \"banana\";
+"({"Object.<anonymous>":function(module,exports,require,__dirname,__filename,global,jest){module.exports = "banana";
```

Comment:
I'm fairly convinced that this shouldn't be happening. Do you have an older version of pretty-format locally? Can you do `npm run clean` and `npm install`? The quotes should definitely be escaped.